src/common/depthmap_toolkit/visualisation.py

```diff
@@ -20,13 +21,59 @@
 SUBPLOT_COUNT = 5
 
 
-def render_pixel(output: object,
+def blur_face(input: np.array, highest: list, dmap: Depthmap) -> np.array:
+
+    # copy values
+    output = np.zeros((dmap.width, dmap.height * SUBPLOT_COUNT, 3))
+    for x in range(dmap.width):
+        for y in range(dmap.height * SUBPLOT_COUNT):
+            output[x][y][:] = input[x][y][:]
+
+    # blur RGB data around face
+    for x in range(dmap.width):
+        for y in range(dmap.height):
+
+            # count distance from the highest child point
+            depth = dmap.parse_depth(x, y)
+            if not depth:
+                continue
+            point = dmap.convert_2d_to_3d_oriented(1, x, y, depth)
+            distance = len(diff(point, highest))
+            if (distance < 0.25):
+
+                # Gausian blur
+                pixel = [0, 0, 0]
+                count = 0
+                step = 5
+                for tx in range(x - step, x + step):
+                    for ty in range(y - step, y + step):
+                        if tx > 0 and ty > 0 and tx < dmap.width and ty < dmap.height:
+                            index = SUBPLOT_RGB * dmap.height + dmap.height - ty - 1
+                            pixel[0] = pixel[0] + input[tx][index][0]
+                            pixel[1] = pixel[1] + input[tx][index][1]
+                            pixel[2] = pixel[2] + input[tx][index][2]
+                            count = count + 1
```

Comment:
If pixel would be a numpy array (like input), this could be simplified
```suggestion
                            pixel = pixel + input[tx][index]
                            count = count + 1
```